### PR TITLE
Jenkins 196 remove orphaned branches

### PR DESCRIPTION
**Pull request by eitzo**

### CODE REVIEW

**Comment by mfuchs** [comment]
From the diff it looks like you might an unrelated commit from Michael is included as well.
Please make sure that only related changes are part of the PR.
if you have any questions on GitHub PRs ping me on slack. 🙂

File: job_dsl/src/main/lib/MultibranchPipelineJobBuilder.groovy [hunk 1]
```diff
@@ -31,7 +31,7 @@ class MultibranchPipelineJobBuilder extends JobBuilder {
 
         job.orphanedItemStrategy {
             discardOldItems {
-                numToKeep(30)
+                numToKeep(1)
```
Comment (mfuchs) [inline job_dsl/src/main/lib/MultibranchPipelineJobBuilder.groovy]: Shouldn't this be 0?
@redeamer what do you think?

Comment (mfuchs) [reply job_dsl/src/main/lib/MultibranchPipelineJobBuilder.groovy]: Ah now I realise (GUI only allows values >= 1). I think we should continue the discussion in the story.

**Comment by redeamer** [comment]
Please create PRs against 'master' in the future and not against some in-dev branches. I will change the base (and hope not to break it ;) )

**Pull request merged**